Library/Homebrew/cmd/tap.rb

```diff
@@ -28,14 +32,16 @@ def install_tap user, repo
     tapd.find_formula { |file| files << file }
     puts "Tapped #{files.length} formula#{plural(files.length, 'e')} (#{tapd.abv})"
 
-    if private_tap?(repouser, repo) then puts <<-EOS.undent
-      It looks like you tapped a private repository. To avoid entering your
-      credentials each time you update, you can use git HTTP credential caching
-      or issue the following command:
+    unless clone_target
```

Comment:
@mikemcquaid Ok, that's easy. I'll just remove the `if` wrapper and leave it as it was originally. Here's my argument against this version though: the whole point of `brew any-tap` was to be able to do more than `brew tap`. For someone who taps numerous private repos, won't it become annoying to get the same reminder message over and over?